### ISSUE TITLE
NPE thrown from Fix Return Type code-action

### ISSUE THREAD

**Issue by rasika** [ballerina-platform/ballerina-lang#29466]
**Description:**
Try to hover over `return self.bar`.

```
class Foo {
    int bar;

    function init() {
        self.bar = 1 ;
    }

    function getBar() {
        return self.bar;
    }
}
```

**Affected Versions:**
2.0.0-SLA3

**Comment by nadeeshaan** [comment]
This issue has been fixed in latest master. Hence closing the  issue

**Comment by IMS94** [comment]
This should be fixed with #29337. Verified with current master and the `change return type to int` code action is suggested.
